main.go

```diff
@@ -15,6 +16,7 @@ import (
 
 	"github.com/briandowns/spinner"
 	systemd "github.com/coreos/go-systemd/v22/dbus"
+	"github.com/redhatinsights/rhc/internal/http"
```

Comment:
This URL is created in the PR when added the http package. It is a way to import an internal package https://www.practical-go-lessons.com/chap-11-packages-and-imports#import-path

Reply:
I see, but the package `http` is part of this repository. Shouldn't we import it using something like `import internal/http`? This looks like that `http` package is part of another repository, and it is not true.

Is there any benefit of using this approach?

Reply:
Yeah, I was wrong in the suggestion. I know that `import "internal/http"` would try to import package from `/usr/lib/golang/src/`. I wanted to suggest to use relative imports.

When you use `import "github.com/redhatinsights/rhc/internal/http"`, then you probably cannot share code on another remote repository e.g. on gitlab or dist-git. You actually can, but then it will not work as expected.

Reply:
I still think that this is issue. Imagine that this code is shared for some reason on some gitlab (notgithub) or fork. It does not matter, but some feature branch is created. Then old code from "github.com/redhatinsights/rhc/internal/http" is still used despite there is new code in feature branch.

Reply:
While this is true, I'd rather preserve the current convention and fix it deliberately in a separate PR than complicate this PR further. The module name of the rhc project is `github.com/redhatinsights/rhc`. We *can* change it to `rhc` if we desire, but we should do this in a separate PR.

Reply:
OK, fixing this in separate PR sounds good.